pints/plot.py

```diff
@@ -198,11 +198,19 @@ def pairwise(chain, kde=False):
     """
     import matplotlib.pyplot as plt
     import numpy as np
-    from scipy import stats
+    import scipy.stats as st
+
+    if thinning is not None:
```

Comment:
Do we really need this? It's not hard for users to just do `pairwise(chain[::4])` instead.